### PR TITLE
Add `.editorconfig` & PR, issue templates

### PR DESCRIPTION
**Pull request by chihsuan**
### Changes proposed in this Pull Request:

- Add `.editorconfig` from WC Core for maintaining consistent coding styles 
- Add PR template (modified from WC Core's template)
- Add issue templates (modified from WC Core's templates)

- [ ] This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).

### How to test the changes in this Pull Request:

Review the changes

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2-enhancement.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+labels: ["type: enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for contributing!
+
+        Please provide us with the information requested in this form. 
+
+        Make sure to look through [existing `type: enhancement` issues](https://github.com/Automattic/wc-calypso-bridge/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+enhancement%22) and [existing `votes needed` issues](https://github.com/Automattic/wc-calypso-bridge/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc+label%3A%22needs%3A+votes%22+) to see whether your idea is already being discussed.
```
Comment (ilyasfoo) [inline .github/ISSUE_TEMPLATE/2-enhancement.yml:13]: `votes needed` label isn't available here. Perhaps we can remove the sentence altogether?

**Review by ilyasfoo — APPROVED** [review]
Excellent, I especially welcome `.editorconfig`. LGTM except minor comment. Pre-approving

**Comment by ilyasfoo** [comment]
>In terms of housekeeping, as we recently introduced Github Actions CI for PRs, is there any particular job that you'd like to run? Perhaps linters, etc.?

@xristos3490 I think JS linter could be great, but since we may have existing files with improper formatting, would it slow down our progress since it may err on unrelated changes?

**Review by xristos3490 — APPROVED** [review]

**Comment by xristos3490** [comment]
> I think JS linter could be great, but since we may have existing files with improper formatting, would it slow down our progress since it may err on unrelated changes?

Yup, I hear that! Let's discuss this at the end of this workstream! We could all do a last visit to ensure the quality for the future (even by limiting the rules of the linter, keeping the bare minimum)!

Also, it would be interesting to see if we can fit a Jest test suite! I saw you have a few in place!

Approved!

File: .github/ISSUE_TEMPLATE/2-enhancement.yml (hunk 1 above)
Comment (chihsuan) [reply .github/ISSUE_TEMPLATE/2-enhancement.yml]: Sure. Removed in 89181a9b726ab11b9f17da5039605b1c9d38b26c

**Pull request merged**